addon/instance-initializers/firebase-settings.ts

```diff
@@ -26,10 +28,28 @@ interface AuthAddonConfig {
   };
 }
 
+interface FunctionsAddonConfig {
+  isCustomSetup?: boolean;
+  emulator?: {
+    hostname: string,
+    port: number,
+  };
+}
+
+interface StorageAddonConfig {
+  isCustomSetup?: boolean;
+  emulator?: {
+    hostname: string,
+    port: number,
```

Comment:
Seems like `connectStorageEmulator()` has an `options` parameter.

Can we add an `options?` typing here similar to `FirestoreAddonConfig` interface above? And then we pass that in `connectStorageEmulator()`.